src/common/guac_rect.h

```diff
@@ -64,6 +64,28 @@ typedef struct guac_common_rect {
 void guac_common_rect_init(guac_common_rect* rect, int x, int y, int width, int height);
 
 /**
+ * Extend the rectangle to be of a size which will when divided by the given
+ * divisor will not produce a remainder.
+ *
+ * The rectangle will be shifted to the left and up and adjusted to fit
+ * within the max bounding rect.
+ *
+ * @param divisor
+ *     The divisor for which the rectangle will be adjusted
+ *
+ * @param rect
+ *     The rectangle to adjust
+ *
+ * @param max_rect
+ *     The bounding area in which the given rect can exist.
+ *
+ * @return
+ *     Zero on success, non-zero on error.
+ */
+int guac_common_rect_adjust(int divisor, guac_common_rect* rect,
```

Comment:
Please clarify.

I understand now what `guac_common_rect_adjust()` is intended to do, but the comment here has some issues:

1. Some of the parameter descriptions are missing periods. This is mostly irksome (as it's inconsistent), but I believe it will also play havoc with some documentation parsing tools.
2. It's not immediately clear what it means to adjust the rectangle to fit a given divisor. If the intent is to expand the rectangle to fit a fixed, square NxN grid anchored at (0, 0), that may be a clearer way to describe this.